public/script.js

```diff
@@ -3096,12 +3097,17 @@ export function createRawPrompt(prompt, api, instructOverride, quietToLoud, syst
         prompt.unshift({ role: 'system', content: systemPrompt });
     }
 
-    // If text completion, convert to text prompt by concatenating all message contents
+    // with Chat Completion, the prefill is an additional assistant message at the end.
+    if (api === 'openai' && prefill) {
+        prompt.push({ role: 'assistant', content: prefill });
+    }
+
+    // if text completion, convert to text prompt by concatenating all message contents and adding the prefill as a promptBias.
     if (api !== 'openai') {
         const joiner = isInstruct ? '' : '\n';
         prompt = prompt.map(message => message.content).join(joiner);
         prompt = api === 'novel' ? adjustNovelInstructionPrompt(prompt) : prompt;
-        prompt = prompt + (isInstruct ? formatInstructModePrompt(name2, false, '', name1, name2, true, quietToLoud) : '\n');  // add last line
+        prompt = prompt + (isInstruct ? formatInstructModePrompt(name2, false, prefill, name1, name2, true, quietToLoud) : '\n');  // add last line
```

Comment:
To make it work without instruct, you just need to...
```suggestion
        prompt = prompt + (isInstruct ? formatInstructModePrompt(name2, false, prefill, name1, name2, true, quietToLoud) : '\n' + prefill);  // add last line
```